compiler/qsc_hir/src/hir.rs

```diff
@@ -355,6 +359,7 @@ impl Display for ItemKind {
                 }
             }
             ItemKind::Ty(name, udt) => write!(f, "Type ({name}): {udt}"),
+            ItemKind::Export(_, export) => write!(f, "Export: {export}"),
```

Comment:
nitpick: is it worth including the ident in the pretty print, similar to how it is for `Type` above?